lib/std/private/jsutils.nim

```diff
@@ -16,5 +16,9 @@ when defined(js):
   func `[]`*(arr: BigUint64Array, i: int): JsBigInt {.importjs: "#[#]".}
   func `[]=`*(arr: Float64Array, i: int, v: float) {.importjs: "#[#] = #".}
 
+
   proc hasJsBigInt*(): bool =
     asm """`result` = typeof BigInt != 'undefined'"""
+
+  proc hasBigUint64Array*(): bool =
```

Comment:
now that we have 2, how about:
```nim
proc hasJsFeature*(name: static string): bool =
  asm """`result` = typeof `name` != 'undefined'"""
```

Reply:
I would like something like when appropriate:
```nim
proc hasJsFeature*(name: someEnum): bool =
 let name = $name 
 asm """`result` = typeof `name` != 'undefined'"""
```

Reply:
a bit controversial because of fwd/backward compatibility issues this will lead to; let's defer this to another PR. this one is LGTM for me